frameworks/hdfs/universe/resource.json

```diff
@@ -3,7 +3,7 @@
     "uris": {
       "jre-tar-gz": "{{jre-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
-      "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.11.0.tar.gz",
+      "hdfs-tar-gz": "https://downloads.mesosphere.io/hdfs/assets/hadoop-2.6.0-cdh5.16.1.tar.gz",
```

Comment:
shouldn't it be `downloads.mesosphere.com` not the `.io`?

Reply:
Fixed.